src/renderers.rs

```diff
@@ -37,45 +37,58 @@ impl Renderer {
             mipmap_filter: wgpu::FilterMode::Nearest,
             lod_min_clamp: 0.0,
             lod_max_clamp: 1.0,
-            compare_function: wgpu::CompareFunction::Always,
+            compare: wgpu::CompareFunction::Always,
         });
 
         // Create uniform buffer
         #[rustfmt::skip]
-        let transform: [f32; 16] = [
+        let mut transform: [f32; 16] = [
             1.0, 0.0, 0.0, 0.0,
             0.0, 1.0, 0.0, 0.0,
             0.0, 0.0, 1.0, 0.0,
             0.0, 0.0, 0.0, 1.0,
         ];
```

Comment:
```suggestion
        let transform: [f32; 16] = [
            1.0, 0.0, 0.0, 0.0,
            0.0, 1.0, 0.0, 0.0,
            0.0, 0.0, 1.0, 0.0,
            0.0, 0.0, 0.0, 1.0,
        ];
        let mut transform_bytes: Vec<u8> = Vec::with_capacity(16*4);
        for f in &transform {
            transform_bytes.extend(f.to_bits().to_ne_bytes().iter().copied());
        }
```

This makes it possible to avoid the unsafe code below.

Reply:
Done, thanks for the suggestion!